test/spec/modules/kargoAnalyticsAdapter_spec.js

```diff
@@ -0,0 +1,54 @@
+import kargoAnalyticsAdapter from 'modules/kargoAnalyticsAdapter.js';
+import { expect } from 'chai';
+import { server } from 'test/mocks/xhr.js';
+let events = require('src/events');
+let constants = require('src/constants.json');
+
+describe('Kargo Analytics Adapter', function () {
+  const adapterConfig = {
+    provider: 'kargoAnalytics',
+    options: {
+      bundleId: '',
+      account: 'kargo',
+    },
+  };
+
+  after(function () {
+    kargoAnalyticsAdapter.disableAnalytics();
+  });
+
+  describe('main test flow', function () {
+    beforeEach(function () {
+      kargoAnalyticsAdapter.enableAnalytics(adapterConfig);
+      sinon.stub(events, 'getEvents').returns([]);
+    });
+
+    afterEach(function () {
+      events.getEvents.restore();
+    });
+
+    it('bid timeout should send one request with auction ID and timeout', function() {
+      const bidTimeout = [
+        {
+          bidId: '2baa51527bd015',
+          bidder: 'kargo',
+          adUnitCode: '/19968336/header-bid-tag-0',
+          auctionId: '66529d4c-8998-47c2-ab3e-5b953490b98f'
+        },
+        {
+          bidId: '6fe3b4c2c23092',
+          bidder: 'kargo',
+          adUnitCode: '/19968336/header-bid-tag-1',
+          auctionId: '66529d4c-8998-47c2-ab3e-5b953490b98f'
+        }
+      ];
+      events.emit(constants.EVENTS.AUCTION_INIT, {
+        timeout: 100
+      });
+      events.emit(constants.EVENTS.BID_TIMEOUT, bidTimeout);
```

Comment:
just a little unclear on the purpose of `bidTimeout` array here

Reply:
@fionasequeira really the only value we care about is `bidder` and `auctionId` - some other tests were using the same array structure, but I'm going to go ahead and remove `bidId` and `adUnitCode` since we're not using them in the code. Thanks!